src/main/java/edu/ksu/canvas/attendance/controller/RosterController.java

```diff
@@ -153,4 +154,8 @@ public ModelAndView deleteAttendance(@PathVariable String sectionId, @ModelAttri
         return page;
     }
 
+    @ExceptionHandler(HttpSessionRequiredException.class)
+    public String handleSessionExpired(){
+        return "NoLtiSession";
+    }
```

Comment:
This code should probably go in the `ExceptionController` class, rather than here.